source/getting-started/core-concepts.md

```diff
@@ -1,58 +1,120 @@
-To get started with Ember.js, there are a few core concepts you
-should understand.
+Before moving to more focused topics, it is important to review several
+high-level Ember.js concepts.
 
 ## Templates
 
-Templates, written in the Handlebars language, describe the user interface of
-your application. In addition to plain HTML, templates can contain expressions,
-like `{{title}}` or `{{author}}`, which take information from a component or
-controller and put it into HTML. They can also contain helpers, like
-`{{#if isAdmin}}30 people have viewed your blog today.{{/if}}.` Finally, they
-can contain components such as a template listing blog posts rendering a
-component for each post.
+Ember.js uses templates to organize the layout of HTML in an application. Most
+templates in an Ember codebase are instantly familiar, and look like any
+fragment of HTML. For example:
+
+```handlebars
+<div>Hi, this is a valid Ember template!</div>
+```
+
+Ember templates use the syntax of [Handlebars](http://handlebarsjs.com)
+templates. Anything that is valid Handlebars syntax is valid Ember syntax.
+
+Each template has a context, and properties you access are read off that context.
+For example this template shows a welcome message with a name:
+
+```handlebars
+<div>Hi {{name}}, this is a valid Ember template!</div>
+```
+
+The mustache of this template (the `{{}}`) refers to some dynamic content.
+Besides properties, you can also use helpers and components inside a mustache.
+
+## Helpers
```

Comment:
Components are where we spent 90% of our time developing in most Ember applications. Already experienced users do not leverage helpers because the learned to avoid a volatile and poorly documented part of the API.

I strongly disagree that these concepts should be kicked off core concept entirely.

In many ways, helpers are simpler to understand that components. Like a bound property, the result in a single value being passed. They also provide a great context to begin understanding nested helper invocation, which is not useful this for core concepts but is very useful later on.